integration-tests/src/tests/client/features/adversarial_behaviors.rs

```diff
@@ -264,12 +264,8 @@ fn test_banning_chunk_producer_when_seeing_invalid_chunk_base(
                         if uses_stateless_validation {
                             // With stateless validation the block usually isn't skipped. Chunk validators
                             // won't send chunk endorsements for this chunk, which means that it won't be
-                            // included in the block at all. The only exception is the first few blocks after
-                            // genesis, which are currently handled in a special way.
-                            // In this test the block with height 2 is skipped.
-                            // TODO(#10502): Properly handle blocks right after genesis, ideally no blocks
-                            // would be skipped when using stateless validation.
-                            this_block_should_be_skipped = height < 3;
+                            // included in the block at all.
+                            this_block_should_be_skipped = false;
```

Comment:
In case `uses_stateless_validation` it looks like we do not even need to run the containing loop.
What about `if height > 1 && !uses_stateless_validation {` and move the comment above?

Reply:
It doesn't look like we can skip the loop since it also updates `invalid_chunks_in_this_block` which is used later.
I've updated this to avoid setting `this_block_should_be_skipped = false` here since it is noop: 10a99ffc28deabba8b398c11de18fc7e76a9e308